examples/bank/bank.go

```diff
@@ -222,20 +250,19 @@ func main() {
 		defer serv.Stop()
 		*dbName = "https://root@" + serv.ServingAddr() + "?certs=test_certs"
 	}
-	// Create a database handle
+	// Create a database handle.
 	db, err := client.Open(*dbName)
 	if err != nil {
 		log.Fatal(err)
 	}
 	bank.db = db
 	// Initialize all the bank accounts.
-	const initCash = 1000
-	bank.initBankAccounts(initCash)
+	initCash := bank.initBankAccounts(1000)
 
 	// Start all the money transfer routines.
 	for i := 0; i < *numParallelTransfers; i++ {
 		// Keep transferring upto 10% of initCash between accounts.
-		go bank.continuousMoneyTransfer(initCash / 10)
+		go bank.continuouslyTransferMoney(initCash / 10)
```

Comment:
initCash should now be the total cash so the transfers are going to keep failing because each transfer amount exceeds the cash in each account.